package.json

```diff
@@ -31,7 +31,7 @@
   "author": "Brian Retterer <brian.retterer@okta.com>",
   "license": "Apache-2.0",
   "devDependencies": {
-    "axios": "^0.21.2",
+    "axios": "^0.26.1",
```

Comment:
I looked over the [changelog](https://github.com/axios/axios/blob/master/CHANGELOG.md) and it seems like the breaking changes do not affect us, but worth grepping through the code and ensuring our uses don't overlap with any breaking changes.